pkgs/development/compilers/roslyn/default.nix

```
@@ -94,20 +44,22 @@ in stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    pkg=$out/lib/dotnet/microsoft.net.compilers.toolset/${packageVersion}
-    mkdir -p $out/bin $pkg
-    unzip -q artifacts/packages/Release/Shipping/Microsoft.Net.Compilers.Toolset.${packageVersion}-dev.nupkg \
-      -d $pkg
+    pkg="$out/lib/dotnet/microsoft.net.compilers.toolset/${version}"
+    mkdir -p "$out/bin" "$pkg"
+
+    unzip -q artifacts/packages/Release/Shipping/Microsoft.Net.Compilers.Toolset.${version}-dev.nupkg \
+      -d "$pkg"
     # nupkg has 0 permissions for a bunch of things
-    chmod -R +rw $pkg
+    chmod -R +rw "$pkg"
 
     makeWrapper ${mono}/bin/mono $out/bin/csc \
       --add-flags "$pkg/tasks/net472/csc.exe"
-    makeWrapper ${mono}/bin/mono $out/bin/vbs \
-      --add-flags "$pkg/tasks/net472/vbs.exe"
+    makeWrapper ${mono}/bin/mono $out/bin/vbc \
+      --add-flags "$pkg/tasks/net472/vbc.exe"
   '';
 
   meta = with lib; {
+    mainProgram = "csc";
     description = ".NET C# and Visual Basic compiler";
     homepage = "https://github.com/dotnet/roslyn";
```

```suggestion
    description = ".NET C# and Visual Basic compiler";
    homepage = "https://github.com/dotnet/roslyn";
    mainProgram = "csc";
```
description is always kinda first